### DISCUSSION TITLE
How do I prevent requiring a change of password for new users?

### DISCUSSION THREAD

**Discussion by TheGorf** [immich-app/immich#7592] [General]
This is a private instance. I don't really give a rats booty about password complexity. I create a user, set the password and be done. But immich apparently requires the user to then change the password on next login. Can that be undone somehow? Maybe I missed it, but I don't see anything like the standard "user must change password" option when creating a user. So I assume its being forced on us by something hardcoded in the app?

**Comment by samholton** [comment]
There is no checking if you reuse the same password, so quick workaround would be to create the user and login as the user and set the password again. I'd be up to adding an option to disable the force password change on new user creation if the project is open to it.

**Reply by samholton** [reply]
Added in https://github.com/immich-app/immich/pull/7598
